packages/swap/src/handlers/__tests__/openSwapDepositChannel.test.ts

```diff
@@ -477,3 +477,114 @@ describe(openSwapDepositChannel, () => {
     ).resolves.not.toThrow();
   });
 });
+
+describe('openSwapDepositChannelSchema', () => {
+  const swapBody = {
+    srcAsset: 'BTC',
+    srcChain: 'Bitcoin',
+    destAsset: 'ETH',
+    destChain: 'Ethereum',
+    destAddress: '0x123',
+    amount: '123',
+    fillOrKillParams: {
+      retryDurationBlocks: 2,
+      refundAddress: '0xa56A6be23b6Cf39D9448FF6e897C29c41c8fbDFF',
+      minPriceX128: '1',
+    },
+  };
+
+  it('handles empty ccmParams strings', () => {
+    expect(() =>
+      openSwapDepositChannelSchema.parse({
+        ...swapBody,
+      }),
+    ).not.toThrow();
```

Comment:
a little less verbose
```suggestion
    expect(openSwapDepositChannelSchema.safeParse({ ...swapBody })).toBe(true);
```